.github/workflows/e2e-ci.yml

```diff
@@ -72,8 +72,7 @@ jobs:
       -
         name: Deploy
         run: |
-          # FIXME this generates a template file, can't copy it into crd/
-          go run pkg/crdgen/main.go > chart/templates/crds.yaml
+          go generate
```

Comment:
This probably don't belong here either, because it's responsibility of the developer to run `go generate`.
I believe we should remove it as well and add a `check-changes` workflow like this: https://github.com/rancher/fleet/blob/master/.github/workflows/check-changes.yml

Reply:
`go generate` removed in https://github.com/rancher/gitjob/pull/352/commits/b68955f1fd75cd3a3e2c34af8fb0e815fb56d420

> I believe we should remove it as well and add a check-changes workflow like this

No, we don't need check-changes here. This is needed in fleet because gijob is a subchart, so we need to check that changes in the gitjob subchart are also made in the gitjob repo.

[validate-ci](https://github.com/rancher/gitjob/blob/master/scripts/validate-ci) already fails if `go generate` output is not pushed to git

Reply:
OK! I wasn't totally aware of that duplicity! So generated code checks are performed only on Drone CI.
Thanks for clarifying!